src/apps/Permissions/AppInstanceLabel.js

```diff
@@ -19,7 +19,20 @@ class AppInstanceLabel extends React.PureComponent {
       <Main>
         <Viewport>
           {({ above }) =>
-            above('medium') && showIcon && <AppIconInRow app={app} />
+            above('medium') &&
```

Comment:
I just started wondering if I should use `above('medium')` instead of `!below('medium')`